src/Entity/Message.php

```diff
@@ -30,6 +31,11 @@ class Message
     #[ORM\Column(type: 'datetime')]
     private DateTime $createdAt;
 
+    public function __construct()
+    {
+        $this->createdAt = new DateTime();
```

Comment:
Must be assigned in the constructor, otherwise reading it would trigger [fatal error](https://3v4l.org/YD2R6#v8.3.11). PHPStan [does not](https://3v4l.org/YD2R6#v8.3.11) detect this problem, but psalm [does](https://psalm.dev/r/e1c5a81ec7).